mocks/routes/rural-payments/person.js

```diff
@@ -51,6 +53,33 @@ export default [
               return notFoundResponse(res)
             }
 
+            return okResponse(res, {
+              _data: [person._data],
+              _page: pagination
+            })
+          }
+        }
+      },
+      {
+        id: 'error',
+        type: 'middleware',
+        options: {
+          middleware: (req, res) => {
+            const body = req.body
+
+            errorToggle = !errorToggle
```

Comment:
Is the intention to have this flip-flop each call? Or should it be moved into the if block so that it only happens once? I suppose if every call now has the retry capability it doesn't matter :thinking: